about.css

```diff
@@ -0,0 +1,134 @@
+@font-face {
+  font-family: 'Lato';
+  src: url('./fonts/Lato-Regular.ttf') format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', serif;
+}
+
+.ab-headline {
+  background: url('./imgs/container-bg.jpeg');
+}
+
+.ab-headline-div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.ab-headline-div h2 {
+  color: #f34e3e;
+  font-size: 20.6px;
+  padding-top: 17px;
+}
+
+.ab-headline-div h1 {
+  color: #f7423f;
+  font-weight: bold;
+  font-size: 32.6px;
+  padding-bottom: 50px;
+  padding-top: 50px;
+}
+
+.ab-headline-div p {
+  background: #fff;
+}
+
+.ab-headline-div span {
+  font-size: 14px;
+  text-align: center;
+  padding: 10px;
+  background: #f7f7f8;
+}
+
+.ab-logo-div {
+  background-color: #fff;
+  padding-top: 30px;
+}
+
+.ab-logo-div h2 {
+  text-align: center;
+  padding: 8px;
+  padding-right: 16px;
+  padding-left: 16px;
+  font-size: 25.6px;
+  font-weight: bold;
+}
+
+.ab-logo-div hr {
+  color: #fa4f40;
+  width: 70px;
+  margin-right: auto;
+  margin-left: auto;
+  padding: 0;
+  line-height: 24px;
+  border-bottom-style: solid;
+  margin-top: 0;
+}
+
+.ab-logo-div p {
+  font-size: 14px;
+  text-align: center;
+  padding: 10px;
+}
+
+.logo-cont img {
+  width: 300px;
+  height: 140px;
+  background: #fff;
+  border: #f7423f solid 4.1px;
+}
```

Comment:
- [ ] Please remove the background image appearing under this section and add horizontal lines separating this section from other sections to match the design.

yours|given
---|---
![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/4597c9ac-fef4-4363-8684-58e2fb2677cf)|![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/f86b3f94-c263-4a54-9780-a0c8c7c5fdb4)